templates/bin_setup.erb

```diff
@@ -8,7 +8,7 @@ set -e
 
 # Set up Ruby dependencies via Bundler
 gem install bundler --conservative
-bundle check || bundle install
+bundle check || bundle install --path=vendor/bundle
```

Comment:
Does Travis set an environment variable when it runs, and if so, could we check that here?